src/exojax/spec/tau1height.py

```diff
@@ -0,0 +1,24 @@
+"""Extract tau=1 height pressure at different wavelengths"""
```

Comment:
Not sure but, looks a bit rare to include "1" in the name of modules. I have no good idea but for instance
- tauinfo (assuming we include more analysis functions in it)
- opacitytools 
-  ... no good idea...

Reply:
i mean i talk about "tau1height.py" just in case

Reply:
Thanks! I'll adopt `opacitytools`.